src/components/Modals/FiatRamps/fiatRampProviders/mtpelerin/utils.ts

```diff
@@ -0,0 +1,115 @@
+import type { AssetId } from '@shapeshiftoss/caip'
+import { adapters } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import { QUOTE_TIMEOUT_MS } from 'packages/swapper/src/constants'
+
+import type { GetQuotesProps, RampQuote } from '../../config'
+import type { MtPelerinQuoteResponse, MtPelerinSellLimitsResponse } from './types'
+
+import MtPelerinLogo from '@/assets/mtpelerin.png'
+import { getMtPelerinFiatCurrencies } from '@/components/Modals/FiatRamps/fiatRampProviders/mtpelerin'
+import { getConfig } from '@/config'
+import { bnOrZero } from '@/lib/bignumber/bignumber'
+
+/**
+ * Get minimum sell limits for a currency from Mt Pelerin API
+ */
+const getMtPelerinSellLimits = async (currency: string): Promise<string | null> => {
+  try {
+    const baseUrl = getConfig().VITE_MTPELERIN_API_URL
+    const response = await axios.get<MtPelerinSellLimitsResponse>(
+      `${baseUrl}/currency_rates/sellLimits/${currency}`,
+      {
+        timeout: QUOTE_TIMEOUT_MS,
+      },
+    )
+    return response.data.limit
+  } catch (e) {
+    console.error('Error fetching Mt Pelerin sell limits:', e)
+    return null
+  }
+}
+
+/**
+ * Get Mt Pelerin quote for crypto-fiat conversion
```

Comment:
Comment suggests this only works one-way. 
```suggestion
 * Get Mt Pelerin quote for crypto<->fiat conversion
```